src/js/main.js

```diff
@@ -64,10 +64,12 @@ export function init(el, context, config) {
     sheetToDomInnerHtml(config.sheetId, sheetName, builder, config.comingSoonSheetName, function callback(resp) {
         const sheetValues = resp.sheets[sheetName][0]; // TODO refactor all instances of `resp.sheets[sheetName][0]` to use this `const`
 
-        var shareFn = share(resp.sheets[sheetName][0].title, window.location);
+        const headline = window.guardian && window.guardian.config.page.headline;
+        const shareText = headline || resp.sheets[sheetName][0].title;
```

Comment:
Yes it is. Though you've made me a little paranoid now, so I'll make this more aggressive.